audio/softsynth/fluidsynth.cpp

```diff
@@ -119,13 +124,49 @@ void MidiDriver_FluidSynth::setStr(const char *name, const char *val) {
 	free(val2);
 }
 
+// Soundfont memory loader callback functions.
+
+static void *SoundFontMemLoader_open(const char *filename)
```

Comment:
It would be nice to follow our code formatting guidelines in this whole added block

Reply:
Whoops I missed that. Done.